lib/configure.js

```diff
@@ -310,42 +302,47 @@ function configure (gyp, argv, callback) {
     // this logic ported from the old `gyp_addon` python file
     var gyp_script = path.resolve(__dirname, '..', 'gyp', 'gyp_main.py')
     var addon_gypi = path.resolve(__dirname, '..', 'addon.gypi')
-    var common_gypi = path.resolve(nodeDir, 'common.gypi')
-    var output_dir = 'build'
-    if (win) {
-      // Windows expects an absolute path
-      output_dir = buildDir
-    }
-    var nodeGypDir = path.resolve(__dirname, '..')
+    var common_gypi = path.resolve(nodeDir, 'include/node/common.gypi')
+    fs.exists(common_gypi, function (exists) {
```

Comment:
Can I suggest doing a `stat` here? `exists` is sorta deprecated, or scheduled for deprecation.

(or just do it sync)